tools/obscuroscan_v2/frontend/src/stores/priceStore.js

```diff
@@ -0,0 +1,43 @@
+import { defineStore } from 'pinia';
+import Config from "@/lib/config";
+
+export const usePriceStore = defineStore({
+    id: 'priceStore',
+    state: () => ({
+        ethPriceUSD: null,
+        loading: false,
+        pollingInterval: 5000,  // 5 seconds
+        timer: null,
+    }),
+    actions: {
+        async fetchCount() {
+            this.loading = true;
+            try {
+                let response = await fetch( 'https://api.coingecko.com/api/v3/simple/price?ids=ethereum&vs_currencies=usd');
```

Comment:
Again, best to use `const` everywhere imo. Just stops people from reading the code from having to wonder if there are shenanigans afoot.

Reply:
Fixed 👍